server/conf/i18n/messages

```diff
@@ -152,23 +152,33 @@ button.select=Select
 button.startApp=Start an application
 # Button to view the applications belonging to an applicant.
 button.viewApplications=View applications
+# Button to view and add clients
+button.viewAndAddClients=View and add clients
 
 # Part of the confirmation banner which appears after a new client has been added to a community-based organization.
 content.clientCreated=Your new client {0} has been added to your client list. You can now begin submitting applications for this client.
+# The tooltip of the email input field
+content.emailTooltip=Add an email address for your client to receive status updates about their application automatically. Without an email, you or your community-based organization will be responsible for communicating updates to your client
 # For each client on the client list, shows the number of applications that have been submitted for them.
 content.numberOfAppSubmitted={0} applications submitted
+# Shows the one application that have been submitted for them.
```

Comment:
suggestion:  I'd suggest a clearer phrasing for this comment:  "Shows that a single application has been submitted on behalf of the client."

Reply:
Good suggestion, added.